test/sql/cluster.sql.in

```diff
@@ -85,3 +85,17 @@ ORDER BY 1,2;
 \set ON_ERROR_STOP 0
 CLUSTER VERBOSE _timescaledb_internal._hyper_1_1_chunk;
 \set ON_ERROR_STOP 1
+
+-- test alter column type on hypertable with clustering
+create table cluster_alter(time timestamp, id text, val int);
+create index idstuff ON cluster_alter USING btree (id ASC NULLS LAST, time);
+
+select create_hypertable('cluster_alter', 'time');
+
+insert into cluster_alter values(now(), '123', 1);
+
+CLUSTER cluster_alter using idstuff;
+
+--attemp the alter table
+ALTER TABLE cluster_alter alter column id type int using id::int;
+
```

Comment:
would add this to the test too:
insert into cluster_alter values ( now(), '245', 1);
----cluster again using the index with the changed type
CLUSTER cluster_alter using idstuff;